.rhcicd/clowdapp-connector-pagerduty.yaml

```
@@ -0,0 +1,205 @@
+---
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: notifications-connector-pagerduty
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: notifications-connector-pagerduty
+  spec:
+    envName: ${ENV_NAME}
+    # To be confirmed, see RHCLOUD-33798
+    # testing:
+    #   iqePlugin: notifications
```

Even though the IQE tests have not been written yet, the plugin already exists and it doesn't hurt to add it here.
```suggestion
    testing:
      iqePlugin: notifications
```